AdobeExtensionSample/readme.md

```diff
@@ -0,0 +1,23 @@
+
+# ForeSee Adobe Extension Sample
+
+## Prerequisites
+
+To run this Sample, you need a Adobe Experience Platform Launch account, and have set it up to use in mobile Apps.  
+
+## Installing
+1.  In the Adobe Launch dashboard, create a new Property for Mobile platform.
+2.  Find and install the ForeSee SDK from the extension category. In the Configuration page, you can choose to enable `Debug Logging` and `Skip Pooling Check`<br/>
+     ![Configuration](https://raw.githubusercontent.com/foreseecode/foresee-sdk-android-samples/MOBILSDK-2750/AdobeExtensionSample/docresources/configuration.png)
+3.  Create a Rule  
+    Set the rule to trigger the ForeSee extension's Action: `ForeSee - Check Eligibility`. This will allow ForeSee SDK to check if the user is eligible for a survey. As below:<br/>
+    ![Rule](https://raw.githubusercontent.com/foreseecode/foresee-sdk-android-samples/MOBILSDK-2750/AdobeExtensionSample/docresources/rule.png)
+4.  Create an Environment.
+5.  In Publishing, create a Library.
+6.  Add the Rule you created, and make sure the ForeSee extension is installed. Save and Build.
+7.  Follow the development environment mobile install instructions.
+8.  In the `SampleApplication` class, replace the AppID in  `MobileCore.configureWithAppID("launch-your-appID")` with your own app ID.
```

Comment:
I haven't been able to get the invite to show and I think it's because I don't understand the publishing part. I ended up publishing my app and pushing it all the way through to production, but I'm not sure that was the right thing to do and I'm also not sure how to undo it. Let's talk about this when you get in today and maybe talk about adding those details to these instructions.

Reply:
Sure. Please feel free to put something in the calendar

Reply:
All good now!

<img width="459" alt="Screen Shot 2019-11-21 at 2 52 33 PM" src="https://user-images.githubusercontent.com/122687/69371867-ac027700-0c6e-11ea-8a91-90d9987a8657.png">